_data-prepper/pipelines/configuration/processors/mutate-event.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: default
+title: Mutate event
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Mutate event processors
+
+Mutate event processors allow you to modify events in Data Prepper. The following processors are available:
+
+* [AddEntries](#addentries): Allows you to add entries to an event.
+* [CopyValues](#copyvalues): Allows you to copy values within an event.
+* [DeleteEntry](#deleteentry): Allows you to delete entries from an event.
+* [RenameKey](#renamekey): Allows you to rename keys in an event.
+* [ConvertEntry](#convertentry): Allows you to convert value types in an event.
+
+## AddEntries
+
+The `AddEntries` processor adds entries to an event.
+
+### Configuration
+
+You can configure the `AddEntries` processor with the following options.
+
+| Option | Required | Description |
+| :--- | :--- | :--- |
+| `entries` | Yes | A list of entries to add to an event. |
+| `key` | Yes | The key of the new entry to be added. Some examples of keys include: `my_key`, `myKey`, and `object/sub_Key`. |
```

Comment:
The clause preceding a colon should always be a complete one, so here you could add "the following" and keep the colon, but in this case it's not even necessary.